cypress/integration/Case01_Create_(Sign_up).js

```diff
@@ -2,8 +2,36 @@
 import StartPage from '../PageObjects/StartPage'
 import SignUpPage from '../PageObjects/SignUpPage'
 import HomePage from '../PageObjects/HomePage'
+import SocialLoginPage from '../PageObjects/SocialLoginPage'
+import userObject from '../fixtures/userObject.json'
+
+describe('Test case 1: login via TorusTestUser and Create temporary user', () => {
+  it('login via google', () => {
+    localStorage.setItem('TorusTestUser', JSON.stringify(userObject))
+    StartPage.open()
+    expect(localStorage.getItem('TorusTestUser')).to.not.be.null
+    SocialLoginPage.googleLink.should('be.visible')
+    cy.wait(1000)
+    SocialLoginPage.googleLink.click()
+    HomePage.profileAvatar.should('be.visible')
+    HomePage.sendButton.should('be.visible')
+    HomePage.optionsButton.click()
+    HomePage.logoutButton.click()
+  })
+
+  it('login via facebook', () => {
+    localStorage.setItem('TorusTestUser', JSON.stringify(userObject))
+    StartPage.open()
+    expect(localStorage.getItem('TorusTestUser')).to.not.be.null
+    SocialLoginPage.facebookLink.should('be.visible')
+    cy.wait(1000)
```

Comment:
could you wait for some changes happened with button (e.g. it enables)?
it would be most exact solution then await for 1sec

Reply:
yes, please avoid waits as much as possible. 
if wait is required then add a comment explaining why

Reply:
@YuryAnanyev

Reply:
I now recheck all the tests and will load them on git